platform/broadcom/sai.mk

```diff
@@ -1,6 +1,6 @@
-LIBSAIBCM_XGS_VERSION = 11.2.16.2
+LIBSAIBCM_XGS_VERSION = 11.2.17.2
 LIBSAIBCM_DNX_VERSION = 11.2.13.1-1
-LIBSAIBCM_XGS_BRANCH_NAME = SAI_11.2.16.2
+LIBSAIBCM_XGS_BRANCH_NAME = SAI_11_2_16_2_CS00012372635
```

Comment:
will this be the final branch name?

Reply:
No, this is a temporary branch. Once we get the final fix from BCM, this will be moved to main branch